plugins/org.yakindu.sct.model.stext.ui/src/org/yakindu/sct/model/stext/ui/help/HelpHoverProvider.java

```diff
@@ -31,50 +32,23 @@
 import org.eclipse.ui.PlatformUI;
 import org.eclipse.ui.editors.text.EditorsUI;
 import org.eclipse.ui.help.IWorkbenchHelpSystem;
+import org.eclipse.xtext.Keyword;
+import org.eclipse.xtext.ui.editor.hover.html.DefaultEObjectHoverProvider;
 import org.eclipse.xtext.ui.editor.hover.html.IXtextBrowserInformationControl;
-import org.yakindu.base.utils.jface.help.HelpHoverProvider;
+import org.yakindu.base.utils.jface.help.AbstractUserHelpDocumentationProvider;
 import org.yakindu.sct.model.stext.ui.internal.STextActivator;
 
 import com.google.inject.Inject;
 import com.google.inject.name.Named;
 
 /**
- * Overrides the loadStylesheet method to load a custom stylesheet from this
- * plugin. The HoverControlCreator is overridden too, because we don't want to
- * have the status line visible. (Not possible to set the bg color of the status
- * line)
- * 
+ * Removes the first line, since the documentation already contains a header.
+ * It also removes the open declaration action
  * 
  * @author andreas muelder - Initial contribution and API
  * 
  */
-@SuppressWarnings("restriction")
-public class CustomCSSHelpHoverProvider extends HelpHoverProvider {
-
-	private String fgStyleSheet = null;
-
-	@Inject(optional = true)
-	@Named("org.eclipse.xtext.ui.editor.hover.XtextEditorHover.font")
-	private String fontSymbolicName = "org.eclipse.jdt.ui.javadocfont"; //$NON-NLS-1$ 
-
-	@Inject
-	@Named("stylesheet")
-	private String styleSheetFileName;
-
-	private HoverControlCreator hoverControlCreator;
-	private PresenterControlCreator presenterControlCreator;
-
-	protected String getStyleSheet() {
-		if (fgStyleSheet == null)
-			fgStyleSheet = loadStyleSheet();
-		String css = fgStyleSheet;
-		if (css != null) {
-			FontData fontData = JFaceResources.getFontRegistry().getFontData(
-					fontSymbolicName)[0];
-			css = HTMLPrinter.convertTopLevelFont(css, fontData);
-		}
-		return css;
-	}
+public class HelpHoverProvider extends DefaultEObjectHoverProvider {
 
 	protected String loadStyleSheet() {
 		URL styleSheetURL = STextActivator.getInstance().getBundle()
```

Comment:
changed could may be simplified:

* HelpHoverProvider remains in jface utils bundle
* but doesn't use STextActivator, instead its own Activator.